### PR TITLE
Update build.md /w stretch rootfs and clarifications

### PR DESCRIPTION
**Pull request by stealthybox**

### CODE REVIEW

File: docs/developer/build.md [hunk 1]
```diff
@@ -68,6 +66,7 @@ Connect your device to your machine over USB, and reboot your device into the bo
 
 ...and flash your build:
 
+    $ cout  # cd $OUT
```
Comment (pdsouza) [inline docs/developer/build.md]: This is unnecessary since fastboot will automatically search `$ANDROID_PRODUCT_OUT` for images.

Comment (stealthybox) [reply docs/developer/build.md]: redacted, thanks :+1:

**Comment by stealthybox** [comment]
@pdsouza, thanks -- fixed :)

What are some good ways to build and flash?

I noticed that `lunch maru_bullhead-userdebug` does a device build which doesn't include things like the radio.
I copied in the vendor blobs, bootstrapped the stretch rootfs, and then ran `mka` in the docker container.

When I do:
- `fastboot -w flashall`
- `fastboot flashall  # on top of a fresh maruos install /w magisk`

My Nexus 5x just bootloops.
Re-flashing TWRP and the maru release zip works.

I found this guide which allowed me to package and sign a zip: https://wiki.lineageos.org/signing_builds.html
The resulting zip was missing things like the radio though.

Do I build the radio/misc stuff with breakfast on the lineage bullhead?
Can I look at the code that does the automated builds somewhere?

This is my first android project, so I'm fumbling through this.
I'll update the dev docs to cover these cases though.

Once I have everything building, I'm going to be modding some kernel configs for docker/k8s support.

**Comment by pdsouza** [comment]
@stealthybox Ah yes, the official vendor blobs from Google are missing things. If you check out the device repo [README](https://github.com/maruos/android_device_lge_bullhead/blob/maru-0.6/README.md) you will see a note on using the excellent [android-prepare-vendor](https://github.com/anestisb/android-prepare-vendor) tool to generate all the blobs you need from the latest factory image. After you copy the generated vendor directory over to `vendor/lge/bullhead` you should not have the bootloop issues anymore. Then just `lunch maru_bullhead-userdebug && mka && fastboot -w flashall`, no need to use `breakfast`. That's probably the most convenient way to build and flash while doing development.

**Comment by stealthybox** [comment]
follow up in:
- https://github.com/maruos/maruos/pull/109
- https://github.com/maruos/android_device_lge_bullhead/pull/3

**Pull request merged**